### DISCUSSION TITLE
What's the difference between downloading from the site and the Mac Store

### DISCUSSION THREAD

**Discussion by UncleDreiAI** [upscayl/upscayl#624] [Q&A]
I don't mind supporting, But i was wondering what was the difference?

**Comment by NayamAmarshe** [comment]
- Easy updates
- Priority Support from me via Email or Telegram
- In future, App store version might come with extra models (from the custom-models repository)

There aren't many differences, however for the price you pay to support the project, I try my best to provide some extra value to you :)

**Reply by UncleDreiAI** [reply]
Done! Thanks for all that you do.

**Comment by TimmTomm8** [comment]
I also would like to know ! :)

**Comment by aaronliu0130** [comment]
I'm pretty sure I've already written a reply here? Dunno what happened

Anyways you get better auto-updates, that's pretty much all

**Reply by NayamAmarshe** [reply]
> I'm pretty sure I've already written a reply here? Dunno what happened
> 
> Anyways you get better auto-updates, that's pretty much all

I had a reply here but I was banned by GitHub (false positive) and my existence from this site disappeared for a while 😅  They took the word vacation too seriously 😬
